index.html

```diff
@@ -35,6 +35,90 @@ <h2 class="green">I’m a software developer! </h2></div>
         <li><a href=""><img src="icons/angellist.svg"></a></li>
       </ul>
     </div>
-      </section>
+    <img src="icons/Disabled.svg" alt="next">
+    <div class="works">
+      <h2>My Recent works</h2>
+      <h2><hr></h2>
+      <div class="container1">
+        <div class="img-placeholder">
+        </div>
+        <div class="text">
+          <h2>Multi-Post Stories Gain+Glory</h2>
+          <ul>
+            <li>Ruby on Rails</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+            <li>HTML</li>
+          </ul>
+          <button type="submit">See project</button>
+        </div>
+      </div>
+      <div class="about">
+        <h2>About me</h2><img  class="right-corner" src="icons/Mask Group.svg" alt="">
+        <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+          don’t hesitate to contact me.</p>
+      <button type="submit">Get my resume</button><img class="left-corner" src="icons/illustration 2-about me mobile.svg" alt="">
+      </div>
+     <div class="containers">
+      <div class="language">
+        <ul>
+          <li><img src="icons/Languages.svg" alt=""></li>
+          <li><h3>Languages</h3></li>
```

Comment:
- [ ] Kindly consider adding The technology categories ("Languages", "Frameworks" and "Skills") in a `ul`, each of them in a `li` as they are also a list 😄